CONTRIBUTING.md

```diff
@@ -119,6 +119,18 @@ mode for you, as well as `requirements_dev.txt` and `plugins/sqlfluff-plugin-exa
 Additionally if a dbt virtual environment was specified, you will also have
 `dbt-core`, `dbt-postgres`, and `plugins/sqlfluff-templater-dbt` available.
 
+#### Developing in Docker
+
+To build a simple interactive Docker container, run the following commands:
+```shell
+make create
+make run
```

Comment:
How would I shell into the container if I want to run commands directly there? Would it be worth adding a `make` target for that?

Have you run the `sqlfluff-templater-dbt` tests in the container to verify they work?